### PR TITLE
Copy oci-umount.conf if it exists

### PR DESCRIPTION
**Pull request by ashcrow**
- Copy oci-umount.conf to the host if it exists in the container
- Port changes from Fedora git to docker-fedora

### CODE REVIEW

**Comment by ashcrow** [comment]
@yuqi-zhang note that a fedora rebuild is not needed with these changes.

File: docker-fedora/Dockerfile [hunk 1]
```diff
@@ -26,5 +28,8 @@ COPY config.json.template service.template tmpfiles.template /exports/
 COPY daemon.json /exports/hostfs/etc/docker/container-daemon.json
 # https://github.com/rhatdan/oci-umount/issues/2
 RUN cp /etc/oci-umount.conf /exports/hostfs/etc
+# Copy config if available
+RUN (test -e /etc/oci-umount.conf && cp /etc/oci-umount.conf /exports/hostfs/etc) || true
```
Comment (yuqi-zhang) [inline docker-fedora/Dockerfile]: Shouldn't the line above this be removed?

Also this change is already in f25, but it *shouldn't* be an issue for later versions (as the file will be present)

File: docker-fedora/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,9 @@ LABEL com.redhat.component="docker" \
 
 RUN dnf install --setopt=tsflags=nodocs -y docker container-storage-setup container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine \
     && rpm -V docker container-storage-setup container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine \
-    && mkdir -p /usr/lib/modules && dnf clean all
+    && mkdir -p /usr/lib/modules \
+    && dnf clean all \
+    && mkdir -p /exports/hostfs/etc/docker
```
Comment (yuqi-zhang) [inline docker-fedora/Dockerfile]: Formatting feels kind of weird, maybe condense with "/usr/lib/modules" into one line?

**Comment by yuqi-zhang** [comment]
Oh wow that's a scary red X, sorry...

**Comment by ashcrow** [comment]
@yuqi-zhang ⬆️

**Comment by yuqi-zhang** [comment]
LGTM

@rh-atomic-bot delegate=giuseppe

**Comment by giuseppe** [comment]
I thought about having here only the last version (rawhide) and keep differences for different versions in the Fedora dist git.  What do you think about it? Since anyway these changes are not going to break on newer systems, it is fine to merge them as well.. LGTM

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 0b3863f

**Comment by yuqi-zhang** [comment]
@giuseppe I think that is the plan

**Pull request closed without merging**